tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1131,3 +1148,536 @@ def record_stream(tensor: torch.Tensor, stream_id: int) -> None:
+        :type b: torch.Tensor, type: fp8
+        :param a_sf: Input scale tensor of shape (P). P is computed by the following formula:
+            P = (div_up(shape_m_4_align * div_up(shape_k, 128) * sizeof(float), 128) * 128)/sizeof(float)
+        :type a_sf: torch.Tensor, type: fp32
+        :param b_sf: Weight scale tensor of shape (w_n, w_k)
+        :type b_sf: torch.Tensor, type: fp32
+
+        :return: Output tensor of shape (M, N)
+        :rtype: torch.Tensor, type: bf16
+        """
+        a, b, a_sf, b_sf, c = inputs
+        l, m, n, k = a.shape[0], a.shape[1], b.shape[1], b.shape[2]
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)